### PR TITLE
Add book class to JavaScript

### PR DESCRIPTION
**Pull request by Bifabig**
- Create a JavaScript class called Book.
- Create methods addBook and removeBook for adding and removing a book to the list.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,72 +1,85 @@
...
-    booksList = JSON.parse(storedBooks);
-    booksList.forEach((book) => {
-      addBook(book);
-    });
+  removeBook(id) {
+    const newArray = JSON.parse(localStorage.getItem('booksList'));
+    newArray.filter(() => this.id !== id);
+
+    const booksId = document.getElementById(id);
+    return booksId.remove();
```
Comment (mikethreels) [inline script.js]: - [x] The remove book function only removes the book from the DOM, it doesn't remove the book from the `booklist` or the localStorage. please make sure to update your book array when removing a book.

**Review by mikethreels — CHANGES_REQUESTED** [review]
Hi Team, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- good job setting up the project.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (rloterh) [reply script.js]: well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Soliman & Biftu,  

Your project is complete! 🏁 There is nothing else to say other than... it's time to merge it 🍾 Congratulations!  🎉 💯 🌟 

## Highlights 🎯 
🎲 Add/remove feature is working as intended!

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._

**You can also consider:**
- Kindly include appropriate instructions in your ReadMe on how to _setup_ and _run_ this app, under the _Getting Started_ section.
- Let's improve the look of our ReadMe by removing the underlined part:
image

Implementing the above change(s) will make your ReadMe/ repository stand out even better! 😉

Cheers..Happy coding! 💻  🍷  🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**